core/templates/pages/story-editor-page/story-editor-page.component.spec.ts

```diff
@@ -301,7 +301,7 @@ describe('Story editor page', function() {
 
   it('should init page on undo redo change applied', () => {
     let mockUndoRedoChangeEventEmitter = new EventEmitter();
-    spyOn(UndoRedoService, 'onUndoRedoChangeApplied').and.returnValue(
+    spyOn(UndoRedoService, 'onUndoRedoChangeApplied$').and.returnValue(
```

Comment:
What does the $ mean?

Reply:
$ is a convention for telling that it is observable. https://angular.io/guide/rx-library#naming-conventions-for-observables